src/resmom/linux/alps.c

```diff
@@ -6685,10 +6685,10 @@ alps_system_KNL(void)
 	alps_engine_query_KNL();
 
 	if (basil_1_7_supported)
-		log_event(PBSEVENT_DEBUG4, PBS_EVENTCLASS_NODE, LOG_DEBUG, __func__,
+		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_NODE, LOG_DEBUG, __func__,
 			"This Cray system supports the BASIL 1.7 protocol.");
 	else {
-		log_event(PBSEVENT_DEBUG4, PBS_EVENTCLASS_NODE, LOG_ERR, __func__,
+		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_NODE, LOG_ERR, __func__,
```

Comment:
@riyazhakki Earlier I had made a request asking for this to be changed (in the code):

> The log message would worry me if I saw it while running on an older Cray OS. Especially since as a user I may not know what version of BASIL is supported on that OS. When I asked around, someone said they'd be wondering how to set the BASIL version. Is the else log message needed? If so, can it be logged at a higher debug level? Actually, a higher debug level for both log messages might be good.

@dilip-krishnan said:

> Done. Set it to DEBUG3

And I said:

> Actually, the code shows PBSEVENT_DEBUG4. And the design doc says PBSEVENT_DEBUG2.

I'm requesting the code should be higher than PBSEVENT_DEBUG2 level.  And that the code and the design match.  Please do the needful.

Reply:
@lisa-altair, these changes were done after providing sign-off on the design document. If we think of making a change to the code now, we would need to change the design document and get the sign-offs again which will delay the progress. Also, we believe that this debug level is good enough. If you feel it is really needed, can we have a separate ticket?

Reply:
@riyazhakki If you acknowledge this as a problem then why not fix it in this pull itself and save additional work that comes with a fresh pull. It looks like a minor change.

Reply:
Hi @lisa-altair  and @arungrover I have made the changes to the code and same updated on the design document. Please review and provide your comment / sign off.